greeting-service/public/index.html

```diff
@@ -16,198 +16,224 @@
-        } else {
-          $('#name-state').html('<h3>Operational state: <span class=\"label label-warning\">FAIL</span></h3>');
-        }
-      }
-
-      function updateCircuitState(isOpen) {
-          if (isOpen) {
-            $('#circuit-state').html('<h3>Circuit breaker state: <span class=\"label label-warning\">OPEN</span></h3>');
-          } else {
-            $('#circuit-state').html('<h3>Circuit breaker state: <span class=\"label label-info\">CLOSED</span></h3>');
-          }
-      }
-
-      // WebSocket connections
-      function connectNameService() {
-        const wsUrlBase = nameServiceUrlBase.replace('http://', 'ws://');
-        const socketUrl = `${wsUrlBase}/name-ws`;
-        const wsocket = new WebSocket(socketUrl);
-        wsocket.onmessage = onMessage;
-        return Promise.resolve(wsocket);
-
-        function onMessage(evt) {
-          if (evt.data === 'state:true') {
-            updateNameServiceState(true);
-          } else if (evt.data === 'state:false') {
-            updateNameServiceState(false);
-          } else {
-            // Request accepted
-            $('#name-requests').prepend('GET /name at ' + evt.data + '\n');
-          }
-        }
-      }
-
-      function connectCircuitService() {
-        const cbWsUrlBase = window.location.origin.replace('http://', 'ws://');
-        const cbWsocket = new WebSocket(`${cbWsUrlBase}/cb-ws`);
-        cbWsocket.onmessage = (evt) => {
-          updateCircuitState(evt.data === 'isOpen:true');
-        };
-        return Promise.resolve(cbWsocket);
-      }
-
-      // UI event handlers
-      $('#invoke').click(function(e) {
-        $.getJSON('/api/greeting', function(res) {
-          $('#greeting-result').prepend(JSON.stringify(res) + '\n');
-        });
-      e.preventDefault();
-      });
-
-      $('#toggle').click(function(e) {
-      var newState = $('#name-state').html().indexOf('FAIL') != -1 ? 'ok' : 'fail';
-      $.ajax({
-            url: nameServiceUrlBase + '/api/state',
-            type: 'PUT',
-            headers: {'Content-Type' : 'application/json'},
-            data: JSON.stringify({state:newState}),
-          });
-      e.preventDefault();
-      });
-
-      $('#clear').click(function(e) {
-        $('#greeting-result').html('');
-        e.preventDefault();
-      });
-
-      $('#clear-requests').click(function(e) {
-      $('#name-requests').html('');
-      e.preventDefault();
+      const app = new Vue({
+        el: "#app",
+        data: {
+          greetingResult: '',
+          nameServiceUrlBase: '',
+          nameState: '',
+          nameRequest: '',
+          circuitState: '',
+        },
+        methods: {
+          initializeServices() {
+            this.initNameServiceLocation()
+              .then(this.initNameServiceState)
+              .then(this.connectNameService)
+              .then(this.initCircuitState)
+              .then(this.connectCircuitService)
+              .catch(alert);
```

Comment:
you can change this to `.catch(console.error);` so that you are able to press F12 key and see the error log, and then click on the error log result to show where the error is happening.

![error1](https://user-images.githubusercontent.com/6443576/193875252-d0e689d6-0457-4f4c-abea-0ae6488fd80f.png)
![error2](https://user-images.githubusercontent.com/6443576/193875257-8ece5ac2-b14c-47ee-b4ea-9df60922d495.png)

Reply:
Thanks @helio-frota for finding the error, I will make the changes